python/servo/devenv_commands.py

```diff
@@ -33,11 +33,32 @@ def cargo(self, params):
         if not params:
             params = []
 
+        self.ensure_bootstrapped()
```

Comment:
Well, we don't really need this (for both `rustc` and `cargo`), since we bootstrap nightly by default. But well, not having it is *confusing* 😄